app/models/airbnb.rb

```diff
@@ -13,4 +13,11 @@ def initialize(search)
     @price          = search[:pricing_quote][:rate][:amount]
     @listing_image = search[:listing][:picture_url]
   end
+
+  def self.find_properties(location)
+    properties = AirbnbService.find_properties(location)
+    properties.map do |property|
+      new(property)
```

Comment:
You're not calling `Airbnb.new(property)` here: is this on purpose? I've never seen just `new(property)` before